backend/src/ml_space_lambda/data_access_objects/app_configuration.py

```diff
@@ -0,0 +1,387 @@
+            ServiceType.TRAINING_JOB.value: self.training_job,
+            ServiceType.TRANSFORM_JOB.value: self.transform_job,
+            ServiceType.HPO_JOB.value: self.hpo_job,
+            ServiceType.ENDPOINT.value: self.endpoint,
+            ServiceType.ENDPOINT_CONFIG.value: self.endpoint_congig,
+            ServiceType.NOTEBOOK.value: self.notebook,
+            ServiceType.MODEL.value: self.model,
+        }
+
+    @staticmethod
+    def from_dict(dict_object: dict) -> EnabledServices:
+        return EnabledServices(
+            dict_object[ServiceType.REALTIME_TRANSLATE.value],
+            dict_object[ServiceType.BATCH_TRANSLATE.value],
+            dict_object[ServiceType.LABELING_JOB.value],
+            dict_object[ServiceType.EMR_CLUSTER.value],
+            dict_object[ServiceType.TRAINING_JOB.value],
+            dict_object[ServiceType.TRANSFORM_JOB.value],
+            dict_object[ServiceType.HPO_JOB.value],
+            dict_object[ServiceType.ENDPOINT.value],
+            dict_object[ServiceType.ENDPOINT_CONFIG.value],
+            dict_object[ServiceType.NOTEBOOK.value],
+            dict_object[ServiceType.MODEL.value],
+        )
+
+
+class EMRConfig:
+    def __init__(self, cluster_sizes: list, auto_scaling: EMRAutoScaling, applications: EMRApplications):
+        self.cluster_sizes = cluster_sizes
+        self.auto_scaling = auto_scaling
+        self.applications = applications
+
+    def to_dict(self) -> dict:
+        return {
+            "clusterSizes": ClusterSize.to_dict(self.cluster_sizes),
+            "autoScaling": self.auto_scaling.to_dict(),
+            "applications": EMRApplications.to_dict(self.applications),
+        }
+
+    @staticmethod
+    def from_dict(dict_object: dict) -> EMRConfig:
+        return EMRConfig(
+            ClusterSize.extract_cluster_sizes(dict_object["clusterSizes"]),
+            EMRAutoScaling.from_dict(dict_object["autoScaling"]),
+            EMRApplications.extract_applications(dict_object["applications"]),
+        )
+
+
+class ClusterSize:
+    def __init__(self, name: str, size: int, master_type: str, core_type: str):
+        self.name = name
+        self.size = size
+        self.master_type = master_type
+        self.core_type = core_type
+
+    @staticmethod
+    def to_dict(cluster_size_list: list):
+        list_of_dicts = []
+        for size in cluster_size_list:
+            list_of_dicts.append(
+                {"name": size.name, "size": size.size, "masterType": size.master_type, "coreType": size.core_type}
+            )
+        return list_of_dicts
+
+    @staticmethod
+    def extract_cluster_sizes(cluster_size_list: list):
+        # Given a list of cluster sizes, parse them into a list of ClusterSize objects
+        cluster_object_list = []
+        for size in cluster_size_list:
+            cluster_object_list.append(ClusterSize(size["name"], size["size"], size["masterType"], size["coreType"]))
```

Comment:
Just want to make sure we're tracking the need to update the EMR creation lambdas since the format of this is changing.